src/Bootstrappers/Integrations/PostgresRLSBootstrapper.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Bootstrappers\Integrations;
+
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Database\DatabaseManager;
+use Stancl\Tenancy\Contracts\TenancyBootstrapper;
+use Stancl\Tenancy\Contracts\Tenant;
+use Stancl\Tenancy\Database\Contracts\TenantWithDatabase;
+
+/**
+ * Set Postgres credentials to the tenant's credentials and use Postgres as the central connection.
+ *
+ * This bootstrapper is intended to be used with Postgres RLS (single-database tenancy).
+ *
+ * @see \Stancl\Tenancy\Commands\CreatePostgresUserForTenants
+ * @see \Stancl\Tenancy\Commands\CreateRLSPoliciesForTenantTables
+ */
+class PostgresRLSBootstrapper implements TenancyBootstrapper
+{
+    protected array $originalCentralConnectionConfig;
+    protected array $originalPostgresConfig;
+
+    public function __construct(
+        protected Repository $config,
+        protected DatabaseManager $database,
+    ) {
+        $this->originalCentralConnectionConfig = $config->get('database.connections.central');
+        $this->originalPostgresConfig = $config->get('database.connections.pgsql');
+    }
+
+    public function bootstrap(Tenant $tenant): void
+    {
+        /** @var TenantWithDatabase $tenant */
+        $this->config->set('database.connections.pgsql.username', $tenant->database()->getUsername() ?? $tenant->getTenantKey());
+        $this->config->set('database.connections.pgsql.password', $tenant->database()->getPassword() ?? 'password');
+
+        $this->config->set('database.connections.central', $this->config->get('database.connections.pgsql'));
```

Comment:
What's the point of setting `central` specifically?

The name of the central connection is configurable: https://github.com/archtechx/tenancy/blob/bd9bbe8b41822f7d157cdb4843c4adc0e7404804/assets/config.php#L135

Same with the `purge('central')` call. The central connection can have a custom name. It's not always `central`.

Reply:
Right, I missed that the name's configurable

Reply:
https://github.com/archtechx/tenancy/pull/1105/commits/5754f8bdc4226d72fdfcc181f346ae19ed3ae431